pkgs/applications/science/biology/blast/default.nix

```diff
@@ -0,0 +1,99 @@
+{ stdenv, fetchurl, zlib, bzip2, perl, cpio, gawk, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "blast";
+  version = "2.9.0";
+
+  src = fetchurl {
+    url = "ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/${version}/ncbi-blast-${version}+-src.tar.gz";
+    sha256 = "0my2rpd1bln05sxnp4c5wk5j5y6yx56vi38pzicjfhh9g8nwr453";
+  };
+
+  sourceRoot = "ncbi-blast-${version}+-src/c++";
+  
+  configureFlags = [ "--without-makefile-auto-update" ];
```

Comment:
I think `--with-dll` is very much necessary here, otherwise, `$out/bin` gets some four times as huge.